src/styles/LessonSide.scss

```diff
@@ -9,18 +9,76 @@
 
 #lesson-side-container {
   background-color: colors.$primary-green;
+  display: flex;
+  flex-direction: column;
   height: 100%;
-  padding: 20px;
+  justify-content: space-between;
+  overflow-y: auto;
   width: 40%;
 }
 
+#turtle-logo {
+  color: colors.$text-white;
+  padding: 5%;
+  //align-items: flex-start;
+  text-align: left;
+}
+
+#turtle-logo-text {
+  display: inline-block;
+  font-family: 'PT Sans', sans-serif;
+  font-size: 3.5em;
+  padding-left: 20px;
+  vertical-align: middle;
+}
+
+#turtle-image {
+  display: inline-block;
+  vertical-align: middle;
+  //float:left;
+}
+
+#lesson-side-header {
+  color: colors.$text-white;
+  font-family: 'PT Sans', sans-serif;
+  font-size: 1.5em;
+  padding-bottom: 2.5%;
+  padding-left: 5%;
+  text-align: left;
+}
+
+#lesson-side-body {
+  color: colors.$text-white;
+  font-family: 'Quattrocento Sans', sans-serif;
+  font-size: 1em;
+  padding-left: 5%;
+  padding-right: 5%;
+  padding-top: 2.5%;
+  text-align: left;
+}
+
+#encouraging-turtle {
+  align-items: flex-end;
+  display: flex;
+  flex-shrink: 5;
+  max-width: 400px;
+  min-width: 100px;
+  padding: 5%;
+  width: calc(35vw);
```

Comment:
The calc is not necessary here (you're not calculating anything). Also this seems like a roundabout way of defining the size of the image. You can just do something like width: calc(100px + 10vw) in a single line. You wouldn't need the flex-shrink, max-width, and min-width.